docs/tutorials/contributing/setting_up_the_environment.rst

```diff
@@ -0,0 +1,32 @@
+Setting up the environment
+==========================
+
+Installing all dependencies
+---------------------------
+
+All testing dependencies can be installed by running::
+
+  $ pip install -r requirements.txt
+
+It is recommended to do this using a virtual environment tool of your choice.
```

Comment:
In the interest of being beginner friendly, should we add in some venv commands here?

Reply:
> In the interest of being beginner friendly, should we add in some venv commands here?

I'm not sure. Depending on the level of beginner what's appropriate? For someone who has just done things using anaconda do we go with a Conda environment (my current setup)? Otherwise do we choose virtualenv, pew, venv (which is relatively standard?), pipenv (which is **really** liked by the people who use it)... ?

Perhaps something like:

```
It is recommended to do this using a virtual environment tool of your choice. For example:

If you use anaconda, create an `axelrod_development` environment by running:

$ conda ...

Otherwise, assuming you have `venv` installed, run:

$ venv ...
```

I'm super open to suggestions although I suspect those who need the documentation instructions might need more explanation as to what/why an virtual env is. Those who don't won't need such instructions as they'll have their own preferred tooling...

Reply:
Having something seems better than nothing and I agree we can't maintain instructions for all the choices. Either `conda` or `venv` are probably fine for complete python newbies. For example take Nik's conda instructions [here](https://github.com/Nikoleta-v3/meta-analysis-of-prisoners-dilemma-tournaments).

Anyone with a strong preference probably doesn't need documentation on this and can use whatever they prefer.

Reply:
In 95e2905 I've gone with `venv`, I started writing an example of using `conda` but it gets "slightly" complicated if you wanted to use the available conda installable libraries... Open to suggestions though :)